nautobot/core/templates/generic/object_list.html

```diff
@@ -173,6 +173,16 @@
                                         </div>
                                     </div>
                                 </div>
+<<<<<<< HEAD
+=======
+                            {% endif %}
+                            {% include table_template|default:'panel_table.html' %}
+
+                            <div class="pull-left noprint" id="bulk-action-buttons">
+                                {% consolidate_bulk_action_buttons %}
+                                {% block bulk_buttons %}{% endblock %}
+
+>>>>>>> origin/develop
```

Comment:
Could use some help resolving this merge conflict

Reply:
Looks like a duplicate of lines 187-193 below?